daemon/container_unix.go

```diff
@@ -640,7 +640,7 @@ func (container *Container) updateNetworkSettings(n libnetwork.Network, ep libne
 		return err
 	}
 
-	if container.hostConfig.NetworkMode == runconfig.NetworkMode("bridge") {
+	if container.hostConfig.NetworkMode == runconfig.NetworkMode("default") {
 		networkSettings.Bridge = container.daemon.config.Bridge.Iface
```

Comment:
Also, how important is this setting ? With upcoming networking features (multi-host and remote plugins), the use for this field is very narrow. is it okay to deprecate it ?